R/BiocManager-pkg.R

```diff
@@ -40,10 +40,10 @@ NULL
 #'
 #' - `"repos"`, `"BiocManager.check_repositories"`,
 #'   `"BiocManager.snapshot"`: URLs of additional repositories for use
-#'   by `BiocManger::install()`. See `?repositories`.
+#'   by [`BiocManager::install()`]. See `?repositories`.
 #'
 #' - `"pkgType"`: The default type of packages to be downloaded and
-#'   installed; see `?install.packages`.
+#'   installed; see [`install.packages`].
```

Comment:
The current formatting indicates how one is to get help on `install.packages` and so is informative across help systems, including terminal-based help systems that don't render links.